semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -153,16 +153,16 @@ let variance_modifier (env : env) (x : CST.variance_modifier) =
 
 let member_modifier (env : env) (x : CST.member_modifier) =
   (match x with
-   | `Over tok -> token env tok (* "override" *)
-   | `Late tok -> token env tok (* "lateinit" *)
+   | `Over tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "override" *)
```

Comment:
I like this keyword. We should promote it in a new KeywordAttr constructor. OCaml also has this keyword.